site/src/theme/components/index.ts

```diff
@@ -0,0 +1,48 @@
+import { ThemeOptions } from "@mui/material";
+
+import { MuiCssBaseline } from "./MuiCssBaseline";
+import { MuiTypography } from "./MuiTypography";
+import { MuiIconButton } from "./MuiIconButton";
+import { MuiPaper } from "./MuiPaper";
+import { MuiButton } from "./MuiButton";
+import { MuiContainer } from "./MuiContainer";
+import { MuiIcon } from "./MuiIcon";
+import { MuiSkeleton } from "./MuiSkeleton";
+import {
+  MuiListItemButton,
+  MuiListItemIcon,
+  MuiListItemText,
+} from "./MuiListItem";
```

Comment:
We’ve grouped a few things in `MuiListItem.ts` and `MuiTabs` and these are the only exceptions at the moment. Perhaps, we should split these files or group other files too? If we need a principle for grouping, we can use official MUI component categories: https://mui.com/components/

Reply:
My thinking behind these groupings was that these are styling overrides for components that are often used together. But I realise this is hard to explain and would easily appear as exceptions. I'll split these into separate files.

Regarding your suggestion of using the categories, I like the idea but fear this will result in rather large files (once we start implementing more custom input designs for instance). Perhaps what we can do instead is organise the component overrides by these groups using directories instead, i.e. we would have `src/theme/components/DataDisplays/MuiListItemButtonThemeOptions.ts`.

What do you think?

Reply:
Yeah it’s hard to define a group based on ”often used together” because ”often” is a vague category. I’m happy with any simple rule that a reader can easily reverse-engineer. If keeping each override in a separate file does make the file structure look too artificial, I guess we can go for that.

Reply:
Sadly I can't seem to see the grouped setup in the changelog. I agree with @benwerner01 though.
I consider components "often used together" as sub-components of base components in our design system. When we have our design system components properly setup and documented (probably using storybook), I think that can aid future contributors in making changes.